### ISSUE TITLE
[DESIGN] Selected Search Filters

### ISSUE THREAD

**Issue by jesicarson** [participedia/api#821]
Ref: #803 

Working on some mockups, here's a screenshot showing a few directions at once:
![Screen Shot 2020-01-08 at 11 55 13 AM](https://user-images.githubusercontent.com/13560530/72011111-fdaa9500-320d-11ea-919e-5479bcb81b5f.png)

**Comment by jesicarson** [comment]
the round chip for selected filters is too similar to the round chip for the filter category drop downs. 
the square chip might work better to differentiate between them. 
challenge of how much category info to include (ie: issues / general issues) etc

**Comment by ascott** [comment]
@jesicarson this is looking great! i'm really liking this direction: 
<img width="631" alt="Screen Shot 2020-01-09 at 10 07 25 AM" src="https://user-images.githubusercontent.com/130878/72092787-e7610f80-32c7-11ea-892b-15b1da3a618c.png">

could you also mock up what it might look like with multiple field filters, like adding a location filter for example?

**Comment by jesicarson** [comment]
Thanks @ascott. See below. Couple challenges, maybe ok tho. 
a) the main filter is not part of the descriptive text (ie: it might other wise say "Issues - General Issues:" and "Location - Countries:")
b) "Countries" is plural but there's only one country selected - I'm assuming we don't want to edit the names of labels to account for plural / singular use? 

![Screen Shot 2020-01-13 at 10 25 27 AM](https://user-images.githubusercontent.com/13560530/72282765-307bd100-35f2-11ea-8f1f-c71b2cda6198.png)

**Comment by jesicarson** [comment]
Yes, I think the above works better than either of these, which I tried in-line:

![Screen Shot 2020-01-13 at 10 55 11 AM](https://user-images.githubusercontent.com/13560530/72283548-d976fb80-35f3-11ea-8494-0cd3cce3228e.png)

![Screen Shot 2020-01-13 at 10 59 26 AM](https://user-images.githubusercontent.com/13560530/72283549-d976fb80-35f3-11ea-806e-82c00ebc3311.png)

**Comment by ascott** [comment]
@jesicarson i think this looks like the best option to me: 
![image](https://user-images.githubusercontent.com/130878/72284105-0d9eec00-35f5-11ea-9520-9b89ebd0f048.png)

i would say let's not include the filter parent/main names (ie: Issues, Location) and only include the field name (ie: General Issues, Countries) and leave the field names as is without taking pluralization into account for now. I think the pluralization as it is actually makes it more clear that the filter could contain multiple values.

**Comment by jesicarson** [comment]
@ascott sounds good to me. Lets run with it.

**Comment by ascott** [comment]
@jesicarson great! @zphingphong i've created a dev issue for these changes here: https://github.com/participedia/api/issues/825

**Comment by jesicarson** [comment]
Shipping #825 to prod for now. 

Future consideration: 

Amber had some design ideas - such as reviewing fashion sites for search filter UI ideas, ie: https://www.revolve.com/clothing/br/3699fc/?navsrc=main which has a wider drop down so more of the check boxes are visible on screen, eliminating the need for the selected search chips. They also use a number to indicate how many filters have been selected for each category. 

![Screen Shot 2020-01-22 at 1 25 52 PM](https://user-images.githubusercontent.com/13560530/72935723-b991b700-3d1a-11ea-9248-d86896010d90.png)

**Comment by jesicarson** [comment]
Another one from Amber https://www.fitnessblender.com/

![Screen Shot 2020-05-20 at 4 25 58 PM](https://user-images.githubusercontent.com/13560530/82507279-acad8600-9ab6-11ea-8ba3-a3384cbd98aa.png)

**Comment by ascott** [comment]
https://www.stocksy.com/

<img width="1676" alt="Screen Shot 2020-05-27 at 1 12 49 PM" src="https://user-images.githubusercontent.com/130878/83067945-57c0c100-a01c-11ea-96a1-7ec2ced2bc27.png">

<img width="1678" alt="Screen Shot 2020-05-27 at 1 12 56 PM" src="https://user-images.githubusercontent.com/130878/83067956-5b544800-a01c-11ea-9bc5-78003ad485c7.png">

**Comment by jesicarson** [comment]
Updated Mockups for Homepage / Search Page re-launch October 2020 (for review with Amber)

OPTION 1 (filter icon button always visible, works with filters overlay as shown on [stage](https://participedia-homepage-2020.herokuapp.com/search?selectedCategory=case))

Search page with no query
![search page 1](https://user-images.githubusercontent.com/13560530/95903967-b8426780-0d4b-11eb-8797-fb1a543a622a.jpg)

Search page with "cases" selected 
![search page 2](https://user-images.githubusercontent.com/13560530/95903970-b8dafe00-0d4b-11eb-961e-be5d8704fcd0.jpg)

Search page with keyword search + filters applied to "cases"
![search page 3](https://user-images.githubusercontent.com/13560530/95903972-b9739480-0d4b-11eb-8741-2b2605767660.jpg)

**Comment by jesicarson** [comment]
OPTION 2 (search filters always visible in left column, scroll independently from results, update available filters for each selected tab)

Search page with no query
![Search Filters - left column 1](https://user-images.githubusercontent.com/13560530/95904545-84b40d00-0d4c-11eb-8d7f-a5eadbbecd2d.jpg)
